<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>status_file</class>
 <widget class="QWidget" name="status_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>85</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>754</width>
    <height>85</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>883</width>
    <height>85</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_31">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_38">
         <property name="minimumSize">
          <size>
           <width>77</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
font: 63 12pt &quot;Open Sans Semibold&quot;;
    color:rgb(0, 44, 65);
}</string>
         </property>
         <property name="text">
          <string>File name:</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_41">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>4</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_39">
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color:rgb(0, 0, 0);
font: 63 12pt &quot;Open Sans Semibold&quot;;
}</string>
         </property>
         <property name="text">
          <string>Moana</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextEditable</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_42">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_6">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar
{
    border: solid grey;
    border-radius: 10px;
	background: rgb(225, 225, 225);
	color: rgb(255, 255, 255);
	font: 87 8pt &quot;Arial Black&quot;;
}

QProgressBar::chunk 
{
    background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(10, 161, 93, 255), stop:1 rgba(0, 44, 65, 255));
    border-radius: 10px;
}</string>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_32">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_33">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_40">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: 63 12pt &quot;Open Sans Semibold&quot;;
    color:rgb(0, 44, 65);
}</string>
           </property>
           <property name="text">
            <string>File type:</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextEditable</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_43">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_41">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color:rgb(0, 0, 0);
		font: 63 12pt &quot;Open Sans Semibold&quot;;

}</string>
           </property>
           <property name="text">
            <string>PNG</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextEditable</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_44">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_34">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_42">
           <property name="minimumSize">
            <size>
             <width>63</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
font: 63 12pt &quot;Open Sans Semibold&quot;;
    color:rgb(0, 44, 65);
}</string>
           </property>
           <property name="text">
            <string>File size:</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextEditable</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_45">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>4</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_43">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color:rgb(0, 0, 0);
	font: 63 12pt &quot;Open Sans Semibold&quot;;
}</string>
           </property>
           <property name="text">
            <string>215 KB</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextEditable</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_46">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
